web/styles/app_components.css

```diff
@@ -446,6 +447,17 @@ div.overlay {
         height: 50%;
         transform: rotate(45deg);
     }
+
+    &.bot-avatar::after {
+        content: " ";
+        background-color: hsl(0deg 0% 17%);
+        position: absolute;
+        bottom: -30%;
+        right: -30%;
+        width: 50%;
+        height: 50%;
+        transform: rotate(45deg);
```

Comment:
We can also avoid this CSS duplication since only `background-color` property is different here.